### PR TITLE
Introduce a metric for obtaining the current database size

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/storages/pull/2230/

## This introduces a breaking change

- [x] No

### CODE REVIEW

**Comment by fridex** [comment]
/hold

...until thoth-station/storages#2230 is released and available in this component

**Comment by pacospace** [comment]
> /hold
> 
> ...until [thoth-station/storages#2230](https://github.com/thoth-station/storages/pull/2230) is released and available in this component

new thoth-storages available :)

File: thoth/metrics_exporter/metrics.py [hunk 1]
```diff
@@ -56,6 +56,8 @@
 pypi_stats = Gauge("thoth_pypi_stats", "Statistics collected from PyPI.", ["stats_type"])
 
 # Thoth Database
+graphdb_size = Gauge("graphdb_size", "database size in bytes", [])
```
[the remark this answers is not in the document]
Comment (fridex) [reply thoth/metrics_exporter/metrics.py:59]: Done in ab3e86b :+1:

**Review by pacospace — APPROVED** [review]
Thanks!! 💯

**Pull request merged**